ginga/Control.py

```diff
@@ -1503,7 +1503,7 @@ def _load_n_switch(imname, path, image_future):
                 # perpetuate the image_future
                 image.set(image_future=image_future, name=imname,
                           path=path)
-                self.fv.gui_do(self.switch, image)
+                self.fv.gui_do(self.switch_image, image)
```

Comment:
Hah, yeah, I debated with myself for a while on whether you really meant `_switch` or `switch_image`. Then I took a bet with 50% chance of winning. Still lost. :ghost: